ckanext/pages/actions.py

```diff
@@ -110,7 +112,7 @@ def _pages_list(context, data_dict):
         pg_row = {'title': pg.title,
                   'content': pg.content,
                   'name': pg.name,
-                  'publish_date': pg.publish_date,
+                  'publish_date': pg.publish_date.isoformat() if pg.publish_date else None,
```

Comment:
fixed another bug in actions.py/_page_update().

/pages was not listing the pages migrated from older version of ckanext-pages, because they didn't have page_type attribute, so all set to None. page_type of None will not be listed in /pages page.